apps/extension/src/shared/rpc-endpoints.ts

```diff
@@ -0,0 +1,20 @@
+import { STAKING_TOKEN_METADATA } from '@penumbra-zone/constants/src/assets';
+
+interface RpcEndpoint {
+  name: string;
+  url: string;
+  imageUrl?: string;
+}
+
+export const RPC_ENDPOINTS: RpcEndpoint[] = [
+  {
+    name: 'Penumbra Labs Testnet RPC',
+    url: 'https://grpc.testnet.penumbra.zone',
+    imageUrl: STAKING_TOKEN_METADATA.images[0]?.svg,
+  },
+  {
+    name: 'Penumbra Labs Testnet Preview RPC',
+    url: 'https://grpc.testnet-preview.penumbra.zone',
+    imageUrl: STAKING_TOKEN_METADATA.images[0]?.svg,
+  },
+];
```

Comment:
Let's move this to `packages/constants`, it probably should be included in the migration of our registry to another repo